app.css

```diff
@@ -267,15 +272,133 @@ footer {
   align-items: center;
   text-align: center;
 }
-.contact-button:hover {
+.cont-button:hover {
   background-color: #4c73d6;
   color: white;
 }
-.contact-button:active {
+.cont-button:active {
   background-color: #4c73d6;
   color: white;
 }
-.contact-button:disabled {
+.cont-button:disabled {
   background-color: #6276a8;
   color: white;
 }
+
+@media screen and (max-width: 720px) {
+  .nav-link {
+    display: none;
+  }
+  .logo-info {
+    display: block;
+  }
+}
+
+@media screen and (min-width: 720px) {
```

Comment:
- [ ] For the desktop version the `min-width` should be from 768px upwards. Kindly fix that.

Reply:
Good job in implementing the changes.